### PR TITLE
 Add --prefix option to build-toolchain.bash

### PR DESCRIPTION
**Pull request by easybe**
Some people might want to install the toolchain to a path like /usr/local/retro68.

### CODE REVIEW

**Comment by autc04** [comment]
Hmmm... this will wipe `/usr/local` if someone says `--prefix=/usr/local`.

**Comment by easybe** [comment]
Oops, that is not good! Any suggestions? I’ll update the PR then.

File: build-toolchain.bash [hunk 1]
```diff
@@ -51,6 +52,9 @@ function usage()
 
 for ARG in $*; do
 	case $ARG in
+		--prefix=*)
+			PREFIX="${ARG#*=}"
```
Comment (easybe) [inline build-toolchain.bash]: Something like `PREFIX="${ARG#*=}/retro68"` would do the trick. Do you have a directory name that you would prefer?

**Comment by autc04** [comment]
... or maybe just refuse to use a non-empty directory when installing outside the build directory?

**Comment by autc04** [comment]
Looks good now.
Of course, there is still the problem that we can't just "build" in a local directory as a regular user and *then* `sudo make install` the whole thing somewhere. But I know no way around that.

**Pull request merged**